nova_vm/src/heap/heap_gc.rs

```diff
@@ -1231,6 +1239,8 @@ fn sweep(agent: &mut Agent, bits: &HeapBits, root_realms: &mut [Option<RealmIden
         if !data_views.is_empty() {
             s.spawn(|| {
                 sweep_heap_vector_values(data_views, &compactions, &bits.data_views);
+                sweep_data_view_side_table_values(data_view_byte_lengths, &compactions.data_views);
```

Comment:
**praise:** Deduplicating the methods is a great idea, and this looks like exactly the right approach! Good job breaking through the wall of "how the fff does this fully home-grown GC thing even work?!" <3